app/views/posts/show.html.erb

```diff
@@ -1,2 +1,17 @@
-<h1>Posts #show</h1>
-<h2>This is the posts show page</h2>
\ No newline at end of file
+<div class="main-container">
+<div class="post-container">
+  <h2><%= @post.title %> by <%= @post.author.name %></h2>
+  <h6>Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %></h6>
+  <p class="post-text"><%= @post.text %></p>
+</div>
+
+<div class="comments-cont">
+  <% if @post.all_comments.count.zero? %>
+  <p>Post has no comments yet</p>
+  <%else%>
+  <% @post.all_comments.each do |comment| %>
+    <p><%= comment.author.name %>: <%= comment.text %></p>
+  <% end %>
+  <% end %>
+</div>
+</div>
```

Comment:
- [ ] The same issue occurs here as well. Kindly change `user_id` to `author_id` in the show action

  ![image](https://user-images.githubusercontent.com/81468675/215783770-56aacde0-d7ee-4f25-980e-7239452c16c4.png)